core_impl/src/main/java/com/google/instrumentation/stats/StatsManagerImplBase.java

```diff
@@ -76,7 +79,11 @@ public View getView(ViewDescriptor viewDescriptor) {
 
   @Override
   StatsContextFactoryImpl getStatsContextFactory() {
-    return statsContextFactory;
+    StatsContextFactoryImpl factory = statsContextFactory;
```

Comment:
The local variable `factory` seems redundant, could just use `statsContextFactory`.